packages/opentelemetry-core/src/common/ConsoleLogger.ts

```diff
@@ -16,9 +16,10 @@
 
 import { Logger } from '@opentelemetry/api';
 import { LogLevel } from './types';
+import { getDefaultLogLevel } from '../platform';
 
 export class ConsoleLogger implements Logger {
-  constructor(level: LogLevel = LogLevel.INFO) {
+  constructor(level: LogLevel = getDefaultLogLevel()) {
```

Comment:
I don't think default is a great name for this because if a user configures a log level it isn't the default. Maybe `getEnvironmentLogLevel` or something? You could even do something like this:
```suggestion
  constructor(level: LogLevel = getEnvironmentLogLevel() ?? LogLevel.INFO) {
```

Reply:
that is starting to make more sense. getEnvLogLvl for browser would just return `undefined` today?

Reply:
yes

Reply:
nullish  coalescer broke tests, had to use or 🤷‍♂️

Reply:
> I don't think default is a great name for this because if a user configures a log level it isn't the default. Maybe `getEnvironmentLogLevel` or something? You could even do something like this:

I am second guessing this approach now since we have to validate `getEnv...() ??/|| LogLevel.Info` in various places (core, tracing and metrics repos) now, whereas if we encompass the "env var set or default" in the platform function we only need to worry about its functionality in the core repo. Does that make sense?

Reply:
As such: https://github.com/open-telemetry/opentelemetry-js/pull/974/commits/71627461ddc147b6031faafd0c715c7257f0864c

Reply:
Null coalescing doesn't work in the browser tests. Probably has to do with the version of ts or configuration used by the webpacking. I'm fine with the new impl.